### DISCUSSION TITLE
how to choose merge partition by manual

### DISCUSSION THREAD

**Discussion by SuperEdison** [apache/doris#24176] [RFC]
I saw there is a merge on write or merge on read by doris.
I am wondering if there are many pieces of partition when I insert some data, and my partition is formatted from yyyymmdd.so how merge those pieces of the same partition?

**Comment by yagagagaga** [comment]
Do you want to merge multiple partitions into one?

**Reply by SuperEdison** [reply]
yes, but it is multiple same partitions, It's like in 2023-09-11, there are many data, and I insert it in batches. so must the Doris merge these data and then to be able to finish this insert?

**Reply by yagagagaga** [reply]
Can you provide more information or give a specific example?
